stdlib/Artifacts/docs/src/index.md

```diff
@@ -0,0 +1,21 @@
+# Artifacts
+
+```@meta
+DocTestSetup = :(using Artifacts)
+```
+
+Starting with Julia 1.6, the artifacts support has moved from `Pkg.jl` to Julia itself.
+Until proper documentation can be added here, you can learn more about artifacts in the
+`Pkg.jl` manual at <https://julialang.github.io/Pkg.jl/v1/artifacts/>.
+
+!!! compat "Julia 1.6"
+    Julia's artifacts API requires at least Julia 1.6. In Julia
+    versions 1.3 to 1.5, you can use `Pkg.Artifacts` instead.
+
+
+```@docs
+Artifacts.artifact_meta
+Artifacts.artifact_hash
+Artifacts.find_artifacts_toml
+Artifacts.@artifact_str
```

Comment:
BTW in case anybody wonders why these four functions and not others: I took the Artifacts API reference in the Pkg.jl documentation and extracted all functions from there which also are in Artifacts.jl, and put them in the above list. That seemed safest: these are functions somebody already decided should appear in a public manual. I think there are a few with docstrings which should *not* appear here (they are rather about implementation details), but perhaps a few more should appear? In any case, I rather don't want to be the person to decide that; but I'll happily add more here if people request it.